parm/config/gfs/config.stage_ic

```diff
@@ -26,6 +26,12 @@ case "${CASE}" in
     export CPL_OCNIC=HR1_refactored
     export CPL_WAVIC=HR1_refactored
     ;;
+  "C1152")
+    export CPL_ATMIC=HR3_refactored
+    export CPL_ICEIC=HR3_refactored
+    export CPL_OCNIC=HR3_refactored
+    export CPL_WAVIC=HR3_C1152_refactored
```

Comment:
HR3 only has ice and ocean ICs. And they are the same resolution as paired with C768, so shouldn't be an issue.
```suggestion
    export CPL_ATMIC=HR2_C1158_refactored
    export CPL_ICEIC=HR3_refactored
    export CPL_OCNIC=HR3_refactored
    export CPL_WAVIC=HR1_refactored
```

Until prototype ICs fix the new pattern, I'll need to make the shadow refactored directory whenever there is new stuff. I've already done this for existing prototype ICs workflow is using. The new pattern is:
```
<name>/YYYMMDDHH/atmos|ice|ocean|wave/*.nc
```
or for ensembles
```
<name>/YYYMMDDHH/mem###/atmos|ice|ocean|wave/*.nc
```

Reply:
@aerorahul now the IC setting in this PR is
C1152")
    export CPL_ATMIC=HR2_C1152_refactored
    export CPL_ICEIC=HR3_refactored
    export CPL_OCNIC=HR3_refactored
    export CPL_WAVIC=HR1_refactored

Reply:
can we close this conversion ?